cockatrice/src/main.cpp

```diff
@@ -97,6 +100,19 @@ bool settingsValid()
         !settingsCache->getPicsPath().isEmpty();
 }
 
+void generateClientID()
```

Comment:
Could you also explain to me how you would call the function and nothing else when the function is inside a file that includes/contains the 2 required components (network and settings)?  I can understand what your saying to a point and fully intend on fixing it,  but im just curious if your stretching things on this one a bit.